gsad/src/gsad_cmd.c

```diff
@@ -98,6 +110,32 @@ cmd_response_data_free (cmd_response_data_t* data)
   g_free (data);
 }
 
+/**
+ * @brief Set allow_caching flag of cmd_response_data_t struct
+ *
+ * @param[in]  data           Command response data struct
+ * @param[in]  allow_caching  allow_caching flag to set
+ */
+void
+cmd_response_data_set_allow_caching (cmd_response_data_t *data,
+                                     int allow_caching)
+{
+  data->allow_caching = allow_caching;
+}
+
+/**
+ * @brief Get allow_caching flag of cmd_response_data_t struct
+ *
+ * @param[in]  data  Command response data struct
+ *
+ * @return The allow_caching flag
+ */
+int
+cmd_response_data_get_allow_caching (cmd_response_data_t *data)
```

Comment:
could you rename this function to cmd_response_data_is_allow_caching to make the boolean character obvious?